CONTRIBUTING.md

```diff
@@ -86,20 +88,18 @@ provided configuration with `pre-commit install`.
 
 Pycharm is an amazing and beautiful python IDE that some of us use to work on this project.
 If you combine it with python black and formatting on save, you will get a very efficient
-workflow.
+workflow. It's also especially efficient for git branching, cherry-picking, committing and pushing.
 
-1. pip install black
-2. Run blackd in a terminal
-3. Install BlackConnect plugin
-4. Set to run python black on save
-5. Set line length to 120
-6. Install the linters in the root directory
+1. Run blackd in a terminal
+2. Install BlackConnect plugin
+3. Set to run python black on save
+4. Set line length to 120
+5. Install the linters in the root directory
 
 ## Testnets and review environments
 
-With the launch of `1.0.0` we will begin running an official `testnet`.
-Prior to the release of `1.1.0` there will be two running test nets. `testnet` and `transaction-beta-testnet`. The `transaction-beta-testnet` testnet will be a beta of the pending 1.1 release, which will enable transactions on the chia blockchain.
-Following the release of `1.1.0`, the official `testnet` will include all changes that have been accepted to the current release branch.
+The current official testnet is testnet10. Look at `chia/util/initial_config.yaml` to see the configuration parameters
+for each testnet. Information on how to use the testnet can be found in the wiki.
```

Comment:
Maybe a link to the wiki?

Reply:
These links change a lot so I rather avoid adding a link

Reply:
Fair, another check to add.  A link checker.